src/vs/code/electron-main/app.ts

```diff
@@ -82,6 +82,7 @@ import { generateUuid } from 'vs/base/common/uuid';
 import { VSBuffer } from 'vs/base/common/buffer';
 import { EncryptionMainService, IEncryptionMainService } from 'vs/platform/encryption/electron-main/encryptionMainService';
 import { ActiveWindowManager } from 'vs/platform/windows/common/windowTracker';
+import { IMainKeyboardLayoutService, MainKeyboardLayoutService } from 'vs/platform/keyboardLayout/electron-main/keyboardLayoutMainService';
```

Comment:
Just a detail, but most of our `main` services are called `I<Kind>MainService`, so I suggest to call it `IKeyboardLayoutMainService`

Reply:
:+1: